### PR TITLE
Submit to Treeherder; bump shared-library version

### PR DESCRIPTION
**Pull request by stephendonner**
@mozilla/firefox-test-engineering / @davehunt r?

### CODE REVIEW

**Comment by stephendonner** [comment]
Ad-hoc build here: https://fx-test-jenkins-dev.stage.mozaws.net:8443/job/treeherder.adhoc/9/

Full console log in plain-text, here: https://gist.github.com/stephendonner/0a7dc2d8af53f3613fc5b743bbce5948

**Comment by rbillings** [comment]
I'm a little out of the loop on this one- is this necessary? This repo is in the middle of being moved to the primary treeherder repo & creating new PRs muddies the water.

**Comment by rbillings** [comment]
I guess a better question is to ask if this is required right away as it will cause Jenkins to fail, or if we can delay the upgrade?

**Comment by stephendonner** [comment]
@rbillings from my standpoint, it's not necessary/urgent (@davehunt can shed more light on the priority), and I certainly don't want to block or make a repo move tougher, so this can wait!

File: tox.ini [hunk 1]
```diff
@@ -8,8 +8,9 @@ passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL SAUCELABS_API_KEY SAUCELABS_USE
 deps = -rrequirements/tests.txt
 commands = pytest \
     --junit-xml=results/{envname}.xml \
-    --html=results/{envname}.html \
-    --log-raw=results/{envname}.log \
+    --html=results/{envname}.html --self-contained-html \
+    --log-raw=results/{envname}.txt \
```
Comment (davehunt) [inline tox.ini]: This filename should be `results/{envname}_raw.txt`

Comment (stephendonner) [inline tox.ini]: Fixed

**Pull request merged**